parsec/interfaces/dtd/insert_function_internal.h

```diff
@@ -228,8 +228,10 @@ struct parsec_dtd_taskpool_s {
     int                          task_window_size;
     int32_t                      task_threshold_size;
     int                          total_tasks_to_be_exec;
-    uint32_t                     local_task_inserted;
-    uint8_t                      function_counter;
+    uint32_t                     local_task_inserted;  /* don't waste an atomic operation
```

Comment:
I don't understand this comment

Reply:
Reminder that we do not need to manipulate this field with atomic operations.